queries.sql

```diff
@@ -127,4 +127,60 @@ SELECT
 FROM
     animals
 GROUP BY species, date_of_birth
-HAVING date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
\ No newline at end of file
+HAVING date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
+
+
+-- Day 3
+
+-- Write queries (using JOIN) to answer the following questions:
+-- What animals belong to Melody Pond?
+
+SELECT name
+FROM animals
+JOIN owners
+ON animals.owner_id = 4;
+
+-- List of all animals that are pokemon (their type is Pokemon).
+
+SELECT animals.name
+FROM animals
+JOIN species
+ON animals.species_id = 1;
```

Comment:
- [ ] Great effort working on your `query multiple tables` project milestone, But the screenshot of this particular query result is missing in your pull request, I would kindly suggest you have all the screenshots of the query result uploaded completely.

Reply:
- For this one i have corrected the query to be the following
SELECT name
FROM animals
WHERE species_id = (SELECT id FROM species WHERE name = 'Pokemon');
![Screenshot 2023-08-17 134121](https://github.com/addisalemseifu/vet-clinic-database/assets/65928536/42dd05bf-92de-4b65-8270-f460cfc6b6ed)